src/FSharp.Data.GraphQL.Server.AspNetCore/GraphQLSubscriptionsManagement.fs

```diff
@@ -0,0 +1,32 @@
+namespace FSharp.Data.GraphQL.Server.AspNetCore
+
+module internal GraphQLSubscriptionsManagement =
```

Comment:
Maybe this way as long as it is a single module?
```suggestion
module internal FSharp.Data.GraphQL.Server.AspNetCore.GraphQLSubscriptionsManagement
```

Reply:
ok.

Reply:
But you need to move all the functions to column 0
That was the idea 🙂

Reply:
Done by yourself, so resolving this conversation.